eventmesh-storage-plugin/eventmesh-storage-mongodb/src/main/java/org/apache/eventmesh/storage/mongodb/constant/MongodbConstants.java

```diff
@@ -24,5 +24,14 @@ public class MongodbConstants {
     public static final String SEQUENCE_COLLECTION_NAME = "pub_sub_seq";
     public static final String SEQUENCE_KEY_FN = "topic";
     public static final String SEQUENCE_VALUE_FN = "value";
+    public static final String DOC_KEY_VERSION = "version";
+    public static final String DOC_KEY_DATA = "data";
+    public static final String DOC_KEY_ID = "id";
+    public static final String DOC_KEY_SOURCE = "source";
+    public static final String DOC_KEY_TYPE = "type";
+    public static final String DOC_KEY_DATA_CONTENT_TYPE = "datacontenttype";
+    public static final String DOC_KEY_SUBJECT = "subject";
```

Comment:
```suggestion
    public static final String CLOUD_EVENT_DOC_VERSION = "version";
    public static final String CLOUD_EVENT_DOC_DATA = "data";
    public static final String CLOUD_EVENT_DOC_ID = "id";
    public static final String CLOUD_EVENT_DOC_SOURCE = "source";
    public static final String CLOUD_EVENT_DOC_TYPE = "type";
    public static final String CLOUD_EVENT_DOC_DATACONTENTTYPE = "datacontenttype";
    public static final String CLOUD_EVENT_DOC_SUBJECT = "subject";
```
There is no need to add `KEY_` in the middle of constants. Considering their usage in both `convertToCloudEvent` and `convertToDocument` methods, the suggested naming is more readable.

Reply:
Thanks for the feedback. Implement in latest commit.